src/domain/service/emails.ts

```diff
@@ -279,6 +280,21 @@ export async function sendReviewEmail(
     );
     emailContent = reviewEmail.html;
     title = `[${updatedApp.appId}] A Revised Application has been Submitted`;
+  } else if (oldApplication.isRenewal) {
+    // send a renewal for review email
+    const reviewEmail = await renderReviewRenewalEmail(
+      updatedApp,
+      {
+        firstName: config.email.reviewerFirstName,
+        lastName: config.email.reviewerLastName,
+      },
+      {
+        baseUrl: config.ui.baseUrl,
+        pathTemplate: config.ui.sectionPath,
+      },
+    );
+    emailContent = reviewEmail.html;
+    title = `[${updatedApp.appId}] A Renewal Application has been Submitted`;
```

Comment:
There is a lot of duplicated code here (vs the other branches of the if/else tree). Can this be simplified?

for example, the content passed to `renderReviewRenewalEmail`, `renderReviewRevisedEmail`, and `renderReviewEmail` are all the same, and then you do the same actions with their outputs.

Reply:
@joneubank i did a little cleanup in the commit below, let me know if that's what you had in mind! the rest of the email functions could probably do with some cleanup now that we got rid of the asynchronous config, but that seems like a story for another pr